Index: app/backend/controllers/MemberController.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/backend/controllers/MemberController.php b/app/backend/controllers/NewsController.php
rename from app/backend/controllers/MemberController.php
rename to app/backend/controllers/NewsController.php
--- a/app/backend/controllers/MemberController.php	
+++ b/app/backend/controllers/NewsController.php	
@@ -2,16 +2,19 @@
 
 namespace backend\controllers;
 
-use app\models\Member;
-use app\models\MemberSearch;
+use app\models\NewsSearch;
+use common\models\News;
+use Yii;
 use yii\web\Controller;
 use yii\web\NotFoundHttpException;
 use yii\filters\VerbFilter;
+use yii\web\UploadedFile;
 
+date_default_timezone_set( "Europe/Kiev");
 /**
- * MemberController implements the CRUD actions for Member model.
+ * NewsController implements the CRUD actions for Member model.
  */
-class MemberController extends Controller
+class NewsController extends Controller
 {
     /**
      * @inheritDoc
@@ -38,7 +41,7 @@
      */
     public function actionIndex()
     {
-        $searchModel = new MemberSearch();
+        $searchModel = new NewsSearch();
         $dataProvider = $searchModel->search($this->request->queryParams);
 
         return $this->render('index', [
@@ -67,10 +70,20 @@
      */
     public function actionCreate()
     {
-        $model = new Member();
+        $model = new News();
 
         if ($this->request->isPost) {
-            if ($model->load($this->request->post()) && $model->save()) {
+            if ($model->load($this->request->post())) {
+                $model->file = UploadedFile::getInstance($model, 'file');
+                $model->created_at = date('y-m-j,h-i-s');
+                $model->updated_at = date('y-m-j,h-i-s');
+                $photoId = uniqid();
+                if ($model->file) {
+                    $model->file->saveAs(Yii::getAlias("@webroot") . '/img/' . $photoId . '_news.png');
+                    $model->image = Yii::$app->request->baseUrl . '/img/' . $photoId . '_news.png';
+                }
+                $model->photo_id = $photoId;
+                $model->save();
                 return $this->redirect(['view', 'id' => $model->id]);
             }
         } else {
@@ -93,8 +106,16 @@
     {
         $model = $this->findModel($id);
 
-        if ($this->request->isPost && $model->load($this->request->post()) && $model->save()) {
+        if ($this->request->isPost && $model->load($this->request->post())) {
+            $model->file = UploadedFile::getInstance($model, 'file');
+            $model->updated_at = date('y-m-j,h-i-s');
+            if ($model->file) {
+                $model->file->saveAs(Yii::getAlias("@webroot") . '/img/' . $model->photo_id . '_news.png');
+                $model->image = Yii::$app->request->baseUrl . '/img/' . $model->photo_id . '_news.png';
+            }
+            $model->save();
             return $this->redirect(['view', 'id' => $model->id]);
+
         }
 
         return $this->render('update', [
@@ -120,12 +141,12 @@
      * Finds the Member model based on its primary key value.
      * If the model is not found, a 404 HTTP exception will be thrown.
      * @param int $id ID
-     * @return Member the loaded model
+     * @return News the loaded model
      * @throws NotFoundHttpException if the model cannot be found
      */
     protected function findModel($id)
     {
-        if (($model = Member::findOne(['id' => $id])) !== null) {
+        if (($model = News::findOne(['id' => $id])) !== null) {
             return $model;
         }
 
Index: app/backend/models/MemberSearch.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/backend/models/MemberSearch.php b/app/backend/models/NewsSearch.php
rename from app/backend/models/MemberSearch.php
rename to app/backend/models/NewsSearch.php
--- a/app/backend/models/MemberSearch.php	
+++ b/app/backend/models/NewsSearch.php	
@@ -2,14 +2,14 @@
 
 namespace app\models;
 
+use common\models\News;
 use yii\base\Model;
 use yii\data\ActiveDataProvider;
-use app\models\Member;
 
 /**
- * MemberSearch represents the model behind the search form of `app\models\Member`.
+ * NewsSearch represents the model behind the search form of `app\models\Member`.
  */
-class MemberSearch extends Member
+class NewsSearch extends News
 {
     /**
      * {@inheritdoc}
@@ -17,8 +17,11 @@
     public function rules()
     {
         return [
-            [['id'], 'integer'],
-            [['name', 'city'], 'safe'],
+            [['header', 'text'], 'required'],
+            [['text'], 'string'],
+            [['created_at', 'updated_at'], 'safe'],
+            [['header', 'image'], 'string', 'max' => 255],
+            [['header'], 'unique'],
         ];
     }
 
@@ -40,7 +43,7 @@
      */
     public function search($params)
     {
-        $query = Member::find();
+        $query = News::find();
 
         // add conditions that should always apply here
 
@@ -61,8 +64,9 @@
             'id' => $this->id,
         ]);
 
-        $query->andFilterWhere(['like', 'name', $this->name])
-            ->andFilterWhere(['like', 'city', $this->city]);
+        $query->andFilterWhere(['like', 'header', $this->header])
+            ->andFilterWhere(['like', 'created at', $this->created_at])
+            ->andFilterWhere(['like', 'updated at', $this->updated_at]);
 
         return $dataProvider;
     }
Index: app/backend/views/member/index.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/backend/views/member/index.php b/app/backend/views/news/index.php
rename from app/backend/views/member/index.php
rename to app/backend/views/news/index.php
--- a/app/backend/views/member/index.php	
+++ b/app/backend/views/news/index.php	
@@ -1,15 +1,16 @@
 <?php
 
+use common\models\News;
 use yii\helpers\Html;
 use yii\helpers\Url;
 use yii\grid\ActionColumn;
 use yii\grid\GridView;
 
 /* @var $this yii\web\View */
-/* @var $searchModel app\models\MemberSearch */
+/* @var $searchModel app\models\NewsSearch */
 /* @var $dataProvider yii\data\ActiveDataProvider */
 
-$this->title = 'Members';
+$this->title = 'News';
 $this->params['breadcrumbs'][] = $this->title;
 ?>
 <div class="member-index">
@@ -17,7 +18,7 @@
     <h1><?= Html::encode($this->title) ?></h1>
 
     <p>
-        <?= Html::a('Create Member', ['create'], ['class' => 'btn btn-success']) ?>
+        <?= Html::a('Create News', ['create'], ['class' => 'btn btn-success']) ?>
     </p>
 
     <?php // echo $this->render('_search', ['model' => $searchModel]); ?>
@@ -27,13 +28,12 @@
         'filterModel' => $searchModel,
         'columns' => [
             ['class' => 'yii\grid\SerialColumn'],
-
-            'id',
-            'name',
-            'city',
+            'header',
+            'created_at',
+            'updated_at',
             [
                 'class' => ActionColumn::className(),
-                'urlCreator' => function ($action, Member $model, $key, $index, $column) {
+                'urlCreator' => function ($action, News $model, $key, $index, $column) {
                     return Url::toRoute([$action, 'id' => $model->id]);
                  }
             ],
Index: app/backend/views/member/_search.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/backend/views/member/_search.php b/app/backend/views/news/_search.php
rename from app/backend/views/member/_search.php
rename to app/backend/views/news/_search.php
--- a/app/backend/views/member/_search.php	
+++ b/app/backend/views/news/_search.php	
@@ -4,22 +4,21 @@
 use yii\widgets\ActiveForm;
 
 /* @var $this yii\web\View */
-/* @var $model app\models\MemberSearch */
+/* @var $model app\models\NewsSearch */
 /* @var $form yii\widgets\ActiveForm */
 ?>
 
-<div class="member-search">
+<div class="news-search">
 
     <?php $form = ActiveForm::begin([
         'action' => ['index'],
         'method' => 'get',
     ]); ?>
 
-    <?= $form->field($model, 'id') ?>
 
-    <?= $form->field($model, 'name') ?>
+    <?= $form->field($model, 'header') ?>
 
-    <?= $form->field($model, 'city') ?>
+    <?= $form->field($model, 'text') ?>
 
     <div class="form-group">
         <?= Html::submitButton('Search', ['class' => 'btn btn-primary']) ?>
Index: app/backend/views/member/view.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/backend/views/member/view.php b/app/backend/views/news/view.php
rename from app/backend/views/member/view.php
rename to app/backend/views/news/view.php
--- a/app/backend/views/member/view.php	
+++ b/app/backend/views/news/view.php	
@@ -4,14 +4,14 @@
 use yii\widgets\DetailView;
 
 /* @var $this yii\web\View */
-/* @var $model app\models\Member */
+/* @var $model common\models\News */
 
-$this->title = $model->name;
+$this->title = $model->header;
 $this->params['breadcrumbs'][] = ['label' => 'Members', 'url' => ['index']];
 $this->params['breadcrumbs'][] = $this->title;
 \yii\web\YiiAsset::register($this);
 ?>
-<div class="member-view">
+<div class="news-view">
 
     <h1><?= Html::encode($this->title) ?></h1>
 
@@ -29,9 +29,11 @@
     <?= DetailView::widget([
         'model' => $model,
         'attributes' => [
-            'id',
-            'name',
-            'city',
+            'image' ,
+            'header',
+            'text',
+            'created_at',
+            'updated_at'
         ],
     ]) ?>
 
Index: app/backend/views/member/create.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/backend/views/member/create.php b/app/backend/views/news/create.php
rename from app/backend/views/member/create.php
rename to app/backend/views/news/create.php
--- a/app/backend/views/member/create.php	
+++ b/app/backend/views/news/create.php	
@@ -3,13 +3,13 @@
 use yii\helpers\Html;
 
 /* @var $this yii\web\View */
-/* @var $model app\models\Member */
+/* @var $model common\models\News */
 
 $this->title = 'Create Member';
-$this->params['breadcrumbs'][] = ['label' => 'Members', 'url' => ['index']];
+$this->params['breadcrumbs'][] = ['label' => 'News', 'url' => ['index']];
 $this->params['breadcrumbs'][] = $this->title;
 ?>
-<div class="member-create">
+<div class="news-create">
 
     <h1><?= Html::encode($this->title) ?></h1>
 
